client/src/app/sidebar/sidebar.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-sidebar',
+  templateUrl: './sidebar.component.html',
+  styleUrls: ['./sidebar.component.css']
+})
+export class SidebarComponent {
+
+  constructor(){
+    this.checkIfMobileSize();
+  }
+  
+  selectedItem: string = 'Home'; 
+  isSidebarOpen : boolean = true;
+  isMobileSize: boolean = false;
```

Comment:
Try to keep selectable items as enums.

Add docs for this.

Reply:
done